NitroxServer/Communication/Packets/Processors/SeaDragonAttackTargetProcessor.cs

```diff
@@ -0,0 +1,13 @@
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Entities;
+
+namespace NitroxServer.Communication.Packets.Processors;
+
+public class SeaDragonAttackTargetProcessor : TransmitIfCanSeePacketProcessor<SeaDragonAttackTarget>
```

Comment:
Can be simplified using primary constructors
```csharp
public class SeaDragonAttackTargetProcessor(
    PlayerManager playerManager,
    EntityRegistry entityRegistry
) : TransmitIfCanSeePacketProcessor<SeaDragonSwatAttack>(playerManager, entityRegistry)
```

I don't put comments on the others processors but same idea (SeaDragonSwatAttackProcessor, SeaDragonGrabExosuitProcessor, RangedAttackLastTargetUpdateProcessor, ...)